packages/shared/lib/core/wallet/utils/buildFoundryOutputData.ts

```diff
@@ -18,19 +18,19 @@ export async function buildFoundryOutputData(
     metadata: IIrc30Metadata,
     accountId: string
 ): Promise<FoundryOutputBuilderParams> {
-    const immutableAliasUnlockCondition = new ImmutableAccountAddressUnlockCondition(
+    const immutableAccountUnlockCondition = new ImmutableAccountAddressUnlockCondition(
         new AccountAddress(api.bech32ToHex(accountId))
```

Comment:
```suggestion
        new AccountAddress(accountId)
```
in `getDepositAddress` we do this, I don't know if it is necessary to do `api.bech32ToHex`